src/main/java/uz/ccrew/flightmanagement/controller/FlightScheduleController.java

```diff
@@ -52,4 +52,12 @@ public ResponseEntity<Response<Page<FlightScheduleDTO>>> getFlightsByAirport(@Pa
         Page<FlightScheduleDTO> result = flightScheduleService.getAllFlightSchedulesByAirportCode(code, page, size);
         return ResponseMaker.ok(result);
     }
+
+    @GetMapping("get-schedules-delayed-and-on-time")
```

Comment:
```suggestion
    @GetMapping("/list/delayed")
``` 
2chisi "/list/ontime"